### DISCUSSION TITLE
Orchid:screen artisan commands with model parameter

### DISCUSSION THREAD

**Discussion by mucan54** [orchidsoftware/platform#1909] [Ideas]
While I create a screen I have to duplicate the methods for crud operations and model info or for while creating resource with orchid-crud there can be a model parameter which will place the the model which we want to use rather then copy and paste it by manually.

**Comment by tabuna** [comment]
Hi. I think this is a great idea. Watch the release https://github.com/orchidsoftware/crud/releases/tag/3.1.0

Auto-detect class for resource name:
```php
php artisan orchid:resource UserResource

// Find App/User or App/Models/User
```

or set options:

```php
php artisan orchid:resource CustomResource -m App\Models\Post
```
